descope/auth.py

```diff
@@ -355,6 +344,30 @@ def _generate_auth_info(self, response_body: dict, refresh_token: str) -> dict:
                 "tenants", {}
             )
 
+        jwt_response["projectId"] = jwt_response.get(SESSION_TOKEN_NAME, {}).get(
```

Comment:
why do we need to fallback to refresh token's project id and user id?

Reply:
the projectID will be the same in both session token and refresh token so we just take it from the first place we found it.. (its not a new code - I just move it from another place).